examples/js/archive.js

```diff
@@ -89,8 +108,10 @@ const renderImages = (fullResImages, url) => {
 // renders thumbnails or images in thumbnail size
 const renderThumbnails = (thumbnails = [], url, fullResImgs) => {
   const imagesToRender = thumbnails || fullResImgs;
+  const paginatedImages = currPagination.paginate(imagesToRender);
```

Comment:
Are these image the current subset after we paginate? Or all images? Maybe we should name them `currentPages` to make it clear that this is a subset?

Reply:
That sounds good, @jywarren thank you!